src/gwproto/data_classes/components/electric_meter_component.py

```diff
@@ -1,8 +1,7 @@
 """ElectricMeterComponent definition"""
 
 from gwproto.data_classes.components.component import Component
-from gwproto.types import ElectricMeterCacGt
-from gwproto.types.electric_meter_component_gt import ElectricMeterComponentGt
+from gwproto.types import ElectricMeterCacGt, ElectricMeterComponentGt
```

Comment:
For sanity we probably should use full absolute import, e.g.,

```
from gwproto.types.electric_meter_component_gt import ElectricMeterCacGt, ElectricMeterComponentGt
```

It sidesteps certain circular import problems. I could be wrong, but I find myself having to add full paths and I've just gotten into the habit of trying to use the full absolute path within my own packages, unless I'm in tests (in which case importing from the base packages should usually work), or I know a package really will always be a leaf. 

Feel free to ignore.  Reviewing the changes it's clear I don't always follow my own advice.